### PR TITLE
Umd support

### PR DESCRIPTION
**Pull request by igierard**
Added UMD support and removed VERSION global variable that was polluting the global namespace. additionally bumped version  number patch. Assumed that the intended previous version number was 0.1.0 (from the dist/bootstral-limit) and not 0.0.1 (from package.json and component.json) so the new version number is 0.1.1. Also added a main declaration to package.json for compatibility with browserify. Also added bower support.

Basically added all the modern (as of this writing) packaging.

### CODE REVIEW

**Comment by igierard** [comment]
not sure why the travis builds are failing. I'm having trouble parsing the output.
